src/components/Settings/Speech/Speech.container.js

```diff
@@ -0,0 +1,169 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import { withStyles } from 'material-ui/styles';
+import Button from 'material-ui/Button';
+import Paper from 'material-ui/Paper';
+import List, { ListItem, ListItemText } from 'material-ui/List';
+import Menu, { MenuItem } from 'material-ui/Menu';
+import { LinearProgress } from 'material-ui/Progress';
+import ArrowDownwardIcon from 'material-ui-icons/ArrowDownward';
+import ArrowUpwardIcon from 'material-ui-icons/ArrowUpward';
+import FastForwardIcon from 'material-ui-icons/FastForward';
+import FastRewindIcon from 'material-ui-icons/FastRewind';
+
+import FullScreenDialog from '../../FullScreenDialog';
+import {
+  MIN_PITCH,
+  MAX_PITCH,
+  INCREMENT_PITCH,
+  MIN_RATE,
+  MAX_RATE,
+  INCREMENT_RATE
+} from './Speech.constants';
+import messages from './Speech.messages';
+
+const styles = theme => ({
+  container: {
+    display: 'flex',
+    position: 'relative',
+    justifyContent: 'center'
+  },
+  icon: {
+    marginLeft: 3,
+    marginRight: 3,
+    width: 20
+  },
+  progress: {
+    paddingLeft: 2,
+    paddingRight: 2,
+    maxWidth: '50%',
+    minWidth: '20%',
+    marginTop: 20
+  }
+});
+
+const getProgressPercent = (value, min, max) =>
+  Math.round((value - min) / (max - min) * 100.0);
+
+const SpeechContainer = ({
```

Comment:
Should be renamed to `SpeechComponent`

Reply:
This was fixed in https://github.com/shayc/cboard/pull/134/commits/1627dac4bbe6678a5f5a5c879fd3ed6991d274e6 too. Sorry again. :sweat_smile: